src/index.ts

```diff
@@ -58,14 +58,20 @@ const createPlugin = (options: ServiceWorkerConfig = {}): AstroIntegration => {
   return {
     name: PKG_NAME,
     hooks: {
-      "astro:config:setup": ({ command, injectRoute, injectScript }) => {
+      "astro:config:setup": ({
+        command,
+        injectRoute,
+        injectScript,
+        config,
+      }) => {
         const autoRegister = options.registration?.autoRegister ?? true;
         if (autoRegister) {
+          const swPath = join(config.base, SW_NAME);
```

Comment:
Does `base` have a default value if it’s not set? If so, what is it?

Reply:
According to these line it will be '/'

https://github.com/withastro/astro/blob/44f7a28728c56c04ac377b6e917329f324874043/packages/astro/src/core/config/schema.ts#L19
https://github.com/withastro/astro/blob/44f7a28728c56c04ac377b6e917329f324874043/packages/astro/src/core/config/schema.ts#L79

Reply:
> Does `base` have a default value if it’s not set? If so, what is it?

According to these line it will be '/'

https://github.com/withastro/astro/blob/44f7a28728c56c04ac377b6e917329f324874043/packages/astro/src/core/config/schema.ts#L19
https://github.com/withastro/astro/blob/44f7a28728c56c04ac377b6e917329f324874043/packages/astro/src/core/config/schema.ts#L79